services/api/service.go

```diff
@@ -690,16 +690,21 @@ func (api *RelayAPI) handleGetPayload(w http.ResponseWriter, req *http.Request)
 	// Get the response - from memory, Redis or DB
 	// note that mev-boost might send getPayload for bids of other relays, thus this code wouldn't find anything
 	getPayloadResp, err := api.datastore.GetGetPayloadResponse(slot, proposerPubkey.String(), blockHash.String())
-	if err != nil {
-		log.WithError(err).Error("failed getting execution payload")
-		api.RespondError(w, http.StatusBadRequest, err.Error())
-		return
-	}
+	if err != nil || getPayloadResp == nil {
+		log.WithError(err).Warn("failed getting execution payload (1/2)")
+		time.Sleep(750 * time.Millisecond)
```

Comment:
What's the default timeout for this call?
I think it'd be fine to retry straight away or wait for some 100ms